wireless_watcher/nodes/watcher_node

```diff
@@ -93,10 +93,9 @@ def main():
 
     while not rospy.is_shutdown():
         try:
-            ip_str = subprocess.check_output(['ip', 'addr', 'show', dev], stderr=subprocess.STDOUT)
-            if re.search(b'^\s*inet\s', ip_str, re.MULTILINE):
-                connected_pub.publish(True)
-        except subprocess.CalledProcessError:
+            with open(os.path.join(SYS_NET_PATH, dev, 'operstate'), 'r') as f:
+                connected_pub.publish(f.read().replace('\n', '') == 'up')
```

Comment:
Suggest `strip()` here instead of the replace.

Also consider using `'rb'` mode and comparing with `b'up'`. Although this file is probably always ascii (and therefore valid unicode), it's best not to presume if you don't have a documented guarantee.

Reply:
Agreed, I will make these changes.

Reply:
Updated.